### DISCUSSION TITLE
Welcome to Friday Discussions!

### DISCUSSION THREAD

**Discussion by avinashkranjan** [avinashkranjan/Friday#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yashpatilofficial** [comment]
I cloned the FRIDAY repo, I am on Flutter 3.10 and Dart 3.0.2, Getting lot of null safety related errors, Is the project configured for lower versions? If yes should the project be upgraded? HELP NEEDED

**Comment by VOLTAR99** [comment]
facing issue in package: import 'package:flutter_gen/gen_l10n/app_localizations.dart';

its showing URI does'nt exist.
Although I have activated flutter_gen package, its still showing me this error.
Any help or suggestions.
